### PR TITLE
Kyrene Remove dead space in user profile section

### PR DESCRIPTION
**Pull request by KyreneAF**
…w 2 inside userProfile

# Description
Screenshot 2024-06-25 at 5 59 01 PM
s # (WBS) 
This bug was only on an owner account

## Related PRS (if any):
No related PRS

## Main changes explained:
Added a height to a div to remove dead space.
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to dashboard→ View profile → Click on each tab ie(Basic information, Teams, Projects)→
6. verify Update Password, Save Changes and cancel button is right under the content rendered with no dead space.
7. verify this new feature works in [dark mode](https://docs.google.com/document/d/11OXJfBBedK6vV-XvqWR8A9lOH0BsfnaHx01h1NZZXfI)

## Screenshots or videos of changes:
Screenshot 2024-06-25 at 3 56 41 PM
Screenshot 2024-06-25 at 3 56 18 PM
Screenshot 2024-06-25 at 3 55 53 PM
Screenshot 2024-06-25 at 3 58 15 PM

## Note:
Include the information the reviewers need to know.

### CODE REVIEW

**Review by kdaga81 — CHANGES_REQUESTED** [review]
image

image

Can we have "Cancel", "Pause", "Set Final Day" to filled color initially rather than getting highlighted while hovering to make it more intuitive during dark mode?

image

Can we have some margin between the last textfield and the buttons in "Volunteer Teams" tab as it looks bit overlapping.

**Review by DiyaWadhwani — CHANGES_REQUESTED** [review]
Hi Kyrene,
I checked out the changes and the spacing on the teams and projects sub-parts is fine.

teams+projectsview

However, the volunteering times section could use a little more margin around the text areas as well as the buttons. The UI looks manageable in light mode but super crammed up in the dark mode.
I would suggest changing the colour of the buttons on the dark mode as well instead of the change in colour when a user hovers over it.

volunteerTimesDark+Light

Kindly review this. Thanks

**Review by Jingyii800 — DISMISSED** [review]
I have tested this PR, the space between the content and the button looks good now in light mode. But the "save changes" button not so obvious in the dark mode.
Screenshot 2024-06-26 142746
Screenshot 2024-06-26 142825
Screenshot 2024-06-26 142816
Screenshot 2024-06-26 142800

**Review by Logeshwari-Renu — DISMISSED** [review]
I have tested the PR as owner and the dead space is removed for each tab in Profile page. The update, save and cancel button were placed at bottom after some space for the "Team" and "Edit History Tab" due to less data but it did not cause any trouble in navigation.

**Review by Trip2310 — DISMISSED** [review]
Hi Kyrene, I have tested this PR by following given steps, it looks good in dark and light mode. Suggestion: Labels looks very occupied to right
It is working perfectly.
Good work
PR#2387B

**Review by HowieMiao — CHANGES_REQUESTED** [review]
Deadspace has been removed and looks good, I also agree with a lot of the proposed changes above, regarding margins for Volunteering Times tab, as well as filled buttons for dark mode.

On top of this, I believe there should be a margin addition between the Save Changes and Cancel button to match the space between Update Password and Save changes, marked in small red indicator at the bottom of the screenshot provided

image

**Review by KaushikMreddy — CHANGES_REQUESTED** [review]
Hi, 

May I know what the changes you've made are? I still see the dead space between the tables and the buttons. Please highlight the changes using a marker in the images. 

- Edit History Tab
image
- Projects Tab
image

**Review by Sarath-3119 — APPROVED** [review]
The changes said in this PR are reflected as expected in the view profile section. There are no dead spaces between updatepassword, save changes and cancel buttons in all the mentioned sub sections.
unnamed (1)
unnamed (2)
unnamed (3)
unnamed (4)
unnamed

**Review by JatinAgrawal94 — CHANGES_REQUESTED** [review]
The sections for teams and edit history still have dead spaces. Rest everything is fine.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/47699187/717406d5-3da8-49dc-8916-f2fa4c0169cf

**Comment by KyreneAF** [comment]
> image image
> Can we have "Cancel", "Pause", "Set Final Day" to filled color initially rather than getting highlighted while hovering to make it more intuitive during dark mode?
> 
> image
> Can we have some margin between the last textfield and the buttons in "Volunteer Teams" tab as it looks bit overlapping.

-I have made the margin changes please review and approve :)

**Comment by KyreneAF** [comment]
@kdaga81 @DiyaWadhwani  I have updated the changes that you have requested. Please review again and approve if you feel they are met :)

**Review by pavanswaroopl — CHANGES_REQUESTED** [review]
Hello,

Everything looks good, but small changes can make the website look better.
1. A little bit of padding between buttons.
2. The save changes button doesn't look obvious in dark mode even when changes are made.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/58074011/e0e3f862-a89a-48f3-a509-2fa3c310faf0

Screenshot 2024-06-28 at 2 02 45 PM

File: src/components/UserProfile/UserProfile.scss [hunk 1]
```diff
@@ -162,6 +162,10 @@
   padding-right: 10px;
 }
 
+.profileEditButtonContainer {
+  margin-top: 20px;
+}
+
```
Comment (Parth-tech) [inline src/components/UserProfile/UserProfile.scss:168]: Hi @KyreneAF,
I was trying to find a reference to `.profileEditButtonContainer` that has been created above. Could you help me locate the use of this css. 
Thanks for helping out!

**Comment by Parth-tech** [comment]
Hi @KyreneAF,
Below I have attached 2 screenshots where I see abit of dead space between the content and the buttons. So I believe that dead space has to be eliminated. Kindly correct me if I have got it wrong. 

Currently a grid/table (i.e rows and columns based) layout is being used hence that leads to the dead space between the table content above and the buttons. Again if I have interpreted incorrectly kindly correct me. 
Thanks.

`Image Reference`
Screenshot 2024-06-28 at 2 33 05 PM
Screenshot 2024-06-28 at 2 33 12 PM

**Comment by IchinnorovTuguldur** [comment]
Hello Kyrene, 

I think spacing looks good to me. nice job on that!!  But under basic information tab there is "pause button". Isn't it supposed to be aligned with set final day button? Looks bit off. 

Screenshot 2024-06-28 at 6 28 55 PM

**Review by nyelmela — DISMISSED** [review]
I checked the PR as an owner and verified that there is no dead space in each tab. The Update Password, Save Changes, and Cancel buttons were exactly under the content, except for Teams, and Edit History, which had less content. Everything else appears to be fine.
image
image
image

**Comment by wang9hu** [comment]
Hi KyreneAF,

I reviewed the request and your changes, and it seems that this extra space is caused by the right tabs sections height trying to match to the left, and the buttons below are outside of these div elements. So my suggestion for this is to include the buttons in the tab sections.
Screenshot 2024-06-28 at 10 11 23 PM
Screenshot 2024-06-28 at 10 11 17 PM
Screenshot 2024-06-28 at 10 11 40 PM

**Comment by wang9hu** [comment]
The dead spaces are removed as originally requested. Good work.

Screenshot 2024-06-29 at 2 21 21 PM

**Comment by one-community** [comment]
Thank you all, merging!

**Pull request merged**